app/serializers/v2/profiles_serializer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class V2::ProfilesSerializer
+  include JSONAPI::Serializer
+  include JSONAPI::ConditionalRelationships
+
+  set_type :profiles
+
+  attributes :assessment_answers
+
+  belongs_to :person, serializer: ::V2::PersonSerializer
+
+  has_one_if_included :person_escort_record, serializer: PersonEscortRecordsSerializer
```

Comment:
can you add the `SUPPORTED_RELATIONSHIPS` here as well

Reply:
It's not needed as we don't have a profiles controller that would make use of the constant.